metricbeat/mb/testing/data/data_test.go

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package testing
+package data
```

Comment:
Mostly for my own curiosity: why was this package change necessary?

Reply:
If I remember correctly, without it there was a circular import when using `_ "github.com/elastic/beats/metricbeat/include"`